pyproject.toml

```diff
@@ -112,7 +112,7 @@ extra-dependencies = [
   "ddtrace",
 
   # Structured logging
-  "structlog",
+  "structlog==24.2.0",
```

Comment:
- I would prefer a less constraining pin such as `"structlog<=24.3.0"`.

- Also, 24.3.0 does not work?

Reply:
It doesn't. I'll  set it to structlog<=24.2.0